client/my-sites/domains/email-providers-upsell/index.jsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
```

Comment:
Location to import `domainAddNew`.
```suggestion
import { domainAddNew } from 'calypso/my-sites/domains/paths';
import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
```